specs/dynamicdependencies/DynamicDependencies.md

```diff
@@ -451,7 +455,7 @@ HRESULT MddRemovePackageDependency(MDD_PACKAGE_DEPENDENCY_CONTEXT context)
 
 Windows informs the Loader when the package graph is defined, but this doesn't help after process creation (and non-packaged processes never have a static package graph).
 
-**Solution:** Tell the loader when the package graph changes.
+The Dynamic Depednencies API informs the loader when the package graph changes.
```

Comment:
Spelling

Reply:
Doh! Did a full pass correcting depedency, depednency and like variants